index.js

```diff
@@ -30,6 +30,9 @@ class Socket extends stream.Duplex {
 
     super(opts)
 
+    this.__objectMode = !!opts.objectMode // streamx is objectMode by default, so implement readable's fuctionality
+    if (opts.objectMode != null) delete opts.objectMode // causes error with ws...
```

Comment:
I'd prefer strict

```suggestion
    if (opts.objectMode !== null) delete opts.objectMode // causes error with ws...
```

Reply:
this checks for undefined too, strict wont check for undef